tqdm/_tqdm.py

```diff
@@ -810,6 +810,12 @@ def __init__(self, iterable=None, desc=None, total=None, leave=True,
         if smoothing is None:
             smoothing = 0
 
+        try:
+            file.write(u'')
```

Comment:
This isn't a great way to check this (though this does look like a bug in tqdm to me).

E.g., the real `sys.stdout` on py2 will happily accept `u""`, but blow up on any actual non-ASCII unicode.

Seems to me that this is another instance of code that should be taking native strings, or else if it insists on writing unicode, yeah, it can't be writing unicode to sys.stdout/sys.stderr on py2.